Readme.md

```diff
@@ -0,0 +1,48 @@
+# Okta API Access Management Sample Resource Server (ASP.NET Core)
+
+The code sample available in this repository demonstrate the use of [Okta API Access Management](http://developer.okta.com/docs/api/resources/oauth2.html) for authorizing access to a backend API using Okta's Authorization Servers.
```

Comment:
nit: demonstrate -> demonstrates

Also, "This code sample" instead of "The code sample available in this repository"